### PR TITLE
Rich Cards sample updated to 4.4.0b1

### PR DESCRIPTION
**Pull request by axelsrz**
Rich Cards sample updated to 4.4.0b1

### CODE REVIEW

File: samples/Rich-Cards-Bot/dialogs/resources/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from . import adaptive_card_example
```
Comment (stevengum) [inline samples/Rich-Cards-Bot/dialogs/resources/__init__.py]: Missing license header

File: samples/Rich-Cards-Bot/main.py [hunk 2]
```diff
@@ -7,245 +7,46 @@
...
+async def messages(req: web.Request) -> web.Response:
     body = await req.json()
     activity = Activity().deserialize(body)
     auth_header = req.headers['Authorization'] if 'Authorization' in req.headers else ''
+    async def aux_func(turn_context):
+        await bot.on_turn(turn_context)
+
     try:
-        return await ADAPTER.process_activity(activity, auth_header, request_handler)
+        await ADAPTER.process_activity(activity, auth_header, aux_func)
```
Comment (stevengum) [inline samples/Rich-Cards-Bot/main.py]: I didn't see an `adapter.onTurnError()` equivalent in the sample, is this in the library's implementation?

Comment (stevengum) [reply samples/Rich-Cards-Bot/main.py]: Example in 4.4 Core Bot JS: https://github.com/microsoft/BotBuilder-Samples/blob/447028bd6fb18cdc81ef19d5e830027ce4826d5e/samples/javascript_nodejs/13.core-bot/index.js#L28-L38

File: samples/Rich-Cards-Bot/main.py [hunk 3]
```diff
@@ -7,245 +7,46 @@
...
-"""Import AdaptiveCard content from adjacent file"""
-from adaptive_card_example import ADAPTIVE_CARD_CONTENT
+
+from dialogs import MainDialog
+from bots import RichCardsBot
 
 APP_ID = ''
 APP_PASSWORD = ''
-PORT = 9000
+PORT = 3978
```
Comment (stevengum) [inline samples/Rich-Cards-Bot/main.py]: Do you want to add the configuration loading from `.yml` (`.yaml`?) to this sample? Or is that still mid-flight?

File: samples/Rich-Cards-Bot/main.py (hunk 2 above)
Comment (axelsrz) [reply samples/Rich-Cards-Bot/main.py]: There is no current functionality for this, I opened this issue on the matter: #167

**Pull request merged**